### ISSUE TITLE
[TextField] line ripple is always centered

### ISSUE THREAD

**Issue by sorvell** [material-components/material-components-web-components#1382]
**Describe the bug**
It does not start at the spot where the input is clicked.

**To Reproduce**
1. Go to https://material-components.github.io/material-components-web-components/demos/textfield/
2. Use the inspector `Animations` tab to slow down animations to 10%.
3. Click on the right side of the first input.
4. Observe that the line animation starts from the center and not the right side.

**Expected behavior**
Line ripple emanates from the clicked spot.

**Additional context**
Compare with https://material-components.github.io/material-components-web-catalog/#/component/text-field

**Comment by asyncLiz** [comment]
I don't think this is an MWC-specific problem, it looks like an upstream issue with MDC

https://jsbin.com/veyusadafo/edit?html,output

The catalog is an older version. It looks like the transform-origin is no longer work since the line-ripple was changed to use an `::after` pseudo element.
